hack/hypershift/package/addon-operator-package.Containerfile

```diff
@@ -1,3 +1,4 @@
 FROM scratch
 
 ADD . /package
+ADD ./hack/hypershift/package/* /package
```

Comment:
I tried to build the package removing the line ```ADD . /package```, the package built is in KBs. Shouldn't the size be in GBs according to our last call?

> Any reason why the files were moved into a single folder?
Since, this is an operator, I was wondering if we should follow the directory structure similar to this: https://github.com/package-operator/examples/tree/main/2_operators/1_start

Reply:
The package image should be in **kilobytes** not gigabytes.

Reply:
With `.` you're essentially copying every file in the addon-operator repository to the package folder and pushing PKO to make sense of it which it couldn't.

Reply:
Got it. I misheard it. My bad.